src/reducer/preferenceSlice.tsx

```diff
@@ -50,6 +52,9 @@ export const preferenceSlice = createSlice({
   name: "preference",
   initialState,
   reducers: {
+    setAppTheme: (state: any, action: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/batnoter/batnoter/pullRequest?prid=9609391)
```suggestion
    setAppTheme: (state: unknown, action: any) => {
```